File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/plagiarized/L4/08/Kasus4L4.ts
/**
 * Converts miles to kilometers.
 * @param miles The number of miles to convert.
 * @returns The number of kilometers.
 */
function convertMilesToKilo(miles: number): number {
    return miles * 1.609;
}

let counter: number = 1;
console.log("Miles\t\tKilometers");
console.log("-------------------------------");

// Use while loop
while (counter <= 10) {
    // Display the results
    console.log(`${counter}\t\t${convertMilesToKilo(counter)}`);
    counter++;
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-04/plagiarized/L4/08/Kasus4L4.py
def convert_miles_to_kilo(miles):
    return miles * 1.609

print("Miles\t\tKilometers")
print("-------------------------------")

counter = 1
while counter <= 10:
    print(f"{counter}\t\t{convert_miles_to_kilo(counter):.3f}")
    counter += 1

